src/components/Main.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react'
+import SearchForm from './SearchForm'
+import DispResults from './DispResults'
+import { Container } from 'react-bootstrap'
+
+export default class Main extends Component {
+  render() {
+    return (
+      <>
+        <Container className="main">
+          <SearchForm queryLocation={this.props.queryLocation} getMap={this.props.getMap} latLon={this.props.latLon} />
+        </Container>
+        <Container>
+          {this.props.showResults &&
+            <DispResults location={this.props.location} mapUrl={this.props.mapUrl} show={this.props.showResults} latLon={this.props.latLon} />
+          }
+        </Container>
+      </>
```

Comment:
Nice job on separating out logic into different components!

Feedback here is really stylistic choice I suppose: If you have more than 3 or so props, feel free to indent for easier readability:

```
<DispResults
   location={this.props.location} 
   mapUrl={this.props.mapUrl} 
   show={this.props.showResults} 
   latLon={this.props.latLon} 
  />
```

Some eslint rules in codebases have line length limits etc. 👍